app/src/main/java/org/mozilla/fenix/addons/AddonPermissionsDetailsFragment.kt

```diff
@@ -6,30 +6,53 @@ package org.mozilla.fenix.addons
 
 import android.net.Uri
 import android.os.Bundle
+import android.view.LayoutInflater
 import android.view.View
+import android.view.ViewGroup
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.navArgs
 import mozilla.components.feature.addons.ui.translateName
 import org.mozilla.fenix.BrowserDirection
 import org.mozilla.fenix.HomeActivity
-import org.mozilla.fenix.R
+import org.mozilla.fenix.databinding.FragmentAddOnPermissionsBinding
 import org.mozilla.fenix.ext.showToolbar
 
 /**
  * A fragment to show the permissions of an add-on.
  */
 class AddonPermissionsDetailsFragment :
-    Fragment(R.layout.fragment_add_on_permissions),
```

Comment:
Curious why we remove the layout id here?

Reply:
I thought I needed to inflate using view binding, but turns out it wasn't needed after all. Updating PR now...